services/api/routers/crud/crud.py

```diff
@@ -1,60 +1,81 @@
 """crud functions"""
+import math
+import pandas as pd
 import pgeocode
-from sqlalchemy import text
-from sqlalchemy.orm import Session
 
+# from logger import LOGGER
+from shared_models.readings_airnow import ReadingsAirnow
+from shared_models.pydantic_models import Location, PollutantEnum, TimeEnum
+from sqlalchemy import select, text
+from sqlalchemy.orm import Session, load_only
 
-def zipcode_to_latlong(zipcode: str):
+
+def zipcode_to_latlong(zipcode: str) -> Location:
     """helper func returns tuple of lat long"""
     geo = pgeocode.Nominatim("us")
     loc = geo.query_postal_code(zipcode)
-    return loc["latitude"], loc["longitude"]
+    if math.isnan(loc["latitude"]):
```

Comment:
maybe this should raise a value error and pass that to the calling function? I dk though, I'm no expert at a exception handling